### DISCUSSION TITLE
Tabs don't work for latest version of theme.

### DISCUSSION THREAD

**Discussion by YconquestY** [vuepress-theme-hope/vuepress-theme-hope#2191] [Q&A] [answered]
I just upgraded to `vuepress@2.0.0-beta.51` and `vuepress-theme-hope@2.0.0-beta.97`. **Tabs in pages are not properly** rendered after running `npm run docs:build`. Below is an example. Is it a bug or my syntax is problematic?

```markdown
# test

::::: tabs

@ tab foo

::: note
This is a `note` container.
:::

@ tab boo

:::: info
This is an `info` container.

::: details
Here are the details.
:::

::::

:::::

```

**Comment by Mister-Hope** [comment, marked as answer]
I need a reproduction. https://vuepress-theme-hope.github.io/v2/md-enhance/guide/tabs.html#demo is working fine

**Reply by YconquestY** [reply]
Sure. [Here](https://github.com/YconquestY/tmp) is a repo for the test page.

**Reply by YconquestY** [reply]
Good afternoon. Any update on this case?

**Reply by Mister-Hope** [reply]
I don't think the reproduction is valid, because Vue press migrates to esm, and it doesn't support cjs config file anymore.

**Reply by YconquestY** [reply]
I just figured out why some visual effects failed: I used the GitHub workflow for VuePress provided by the official website rather than the theme. Compatibility issues occur when using a different deployment scheme.
